librz/core/cmd_descs/cmd_analysis.yaml

```diff
@@ -996,6 +996,168 @@ commands:
           - name: number
             type: RZ_CMD_ARG_TYPE_RZNUM
             optional: true
+  - name: ag
+    summary: Analysis graph commands
+    details:
+      - name: "Formats"
+        entries:
+          - text: "<blank>"
+            comment: "Ascii art"
+          - text: "*"
+            comment: "rizin commands"
+          - text: "d"
+            comment: "Graphviz dot"
+          - text: "g"
+            comment: "Graph Modelling Language (gml)"
+          - text: "j"
+            comment: "json ('J' for formatted disassembly)"
+          - text: "k"
+            comment: "SDB key-value"
+          - text: "t"
+            comment: "Tiny ascii art"
+          - text: "v"
+            comment: "Interactive ascii art"
+    subcommands:
+      - name: aga
+        summary: Data reference graph
+        cname: analysis_graph_dataref
+        args:
+          - name: format
+            type: RZ_CMD_ARG_TYPE_STRING
```

Comment:
you could use the CHOICE type and specify the choices as well, so that users can also more easily autocomplete.

Reply:
Found a bug in CHOICE

![image](https://user-images.githubusercontent.com/24983369/184955151-3c69d143-4740-48a7-8db1-c80c0e3483e5.png)
![image](https://user-images.githubusercontent.com/24983369/184955699-3a820443-5ddd-4b75-8a55-405a3b727c59.png)

Reply:
Always get an `Address boundary error`, when trying to trigger autocompletion

Reply:
@ret2libc could you please take a look at this when you have time?

Reply:
Will check